base/src/main/java/com/smartdevicelink/proxy/rpc/ImageResolution.java

```diff
@@ -97,6 +97,9 @@ public ImageResolution(@NonNull Integer resolutionWidth, @NonNull Integer resolu
     }
     
     public void setResolutionWidth(@NonNull Integer resolutionWidth) {

```

Comment:
For this method and the corresponding height method, can we please add javadocs explaining why the input may be modified?